src/main/java/com/lambdaschool/wellness/model/User.java

```diff
@@ -43,11 +49,16 @@ public String getLname() {
     // this.lname = lname;
     // }
 
-    public String getFullname() {
-        return this.fname + " " + this.lname;
+    public String getFullname()
+    {
+        return this.fullname = lname + " " + fname;
```

Comment:
Should fullname be first name then last name?

Reply:
yeah i can change it.

Reply:
Once this is changed it looks good to me